### DISCUSSION TITLE
Add More Pokemon Details

### DISCUSSION THREAD

**Discussion by DayManAlt** [jynnie/soullocke#20] [Ideas]
Maybe there could be like an option to make it so you can add the levels, movesets, nature, and ability of both the pokemon.

**Comment by jynnie** [comment]
Hi! Thanks for the suggestion! I purposefully omitted those features, as they tend to over-complicate a lot of Nuzlocke trackers. Hopefully, you can use the built-in Notes feature to keep track of that.

**Comment by DayManAlt** [comment]
Ok, thank you for responding. I do now realize that the notes could be used
for this but I didn't see it before so thank you for that. :)

On Wed, Apr 5, 2023 at 12:14 AM jynnie ***@***.***> wrote:

> Hi! Thanks for the suggestion! I purposefully omitted those features, as
> they tend to over-complicate a lot of Nuzlocke trackers. Hopefully, you can
> use the built-in Notes feature to keep track of that.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jynnie/soullocke/discussions/20#discussioncomment-5527605>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A62Z5QNR7LI3VQZJYLFMKHLW7TWR3ANCNFSM6AAAAAAWMRZRYQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 
_This email originated from a student account managed by the North 
Kingstown School Department.  If you believe it is being used 
inappropriately, please ***@***.*** 
***@***.***>._
